app/views/green_lanes/category_assessments/create.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= page_header "Search results for &lsquo;#{h @category_assessments_search.commodity_code}&rsquo;".html_safe %>
```

Comment:
`html_safe` is often best avoided - I can see the code sanitizes the comm code but in general its better to not add `html_safe` to codebases because it normalises usage and can lead to sanitization issues creeping in. 

One alternative is to use `sanitize`

```ruby
page_header sanitize "some code#{ @some.variable} sdfsldk"
```

Or you could just cut and past the the raw characters in (or use utf8 codes) and let ERB escape them

```ruby
page_header "Something ‘#{@some.variable}’"
```

Or do we really need the wrapping quote characters? I'd argue not

```ruby
page_header "Something #{@some.variable}"
```

Reply:
Removed quote